aptos-move/e2e-move-tests/src/tests/function_value_capture_option.rs

```diff
@@ -0,0 +1,367 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+//! Tests for capturing option type in function values.
+
+use crate::{assert_success, tests::common, MoveHarness};
+use aptos_framework::BuildOptions;
+use aptos_package_builder::PackageBuilder;
+use aptos_transaction_simulation::Account;
+use aptos_types::{
+    on_chain_config::FeatureFlag,
+    transaction::{ExecutionStatus, TransactionStatus},
+};
+use claims::assert_matches;
+use move_core_types::account_address::AccountAddress;
+
+#[test]
+fn test_vm_value_fv_capture_option_1() {
+    let mut h = MoveHarness::new();
+    let acc = h.new_account_at(AccountAddress::from_hex_literal("0x99").unwrap());
+
+    // test directly capture option type
```

Comment:
Can we decouple publish from testing? This will make tests smaller, so we do not rely on init module semantics: 1_ publish the code, then enable disable features and run some entry function?